react/features/youtube-player/components/native/YoutubeLargeVideo.js

```diff
@@ -0,0 +1,256 @@
+     *
+     * @private
+     */
+    _onWideScreenChanged: Function,
+
+    /**
+     * The id of the participant sharing the video.
+     *
+     * @private
+     */
+    _ownerId: string,
+
+    /**
+     * The height of the screen.
+     *
+     * @private
+     */
+    _screenHeight: number,
+
+    /**
+     * The width of the screen.
+     *
+     * @private
+     */
+    _screenWidth: number,
+
+    /**
+     * Seek time in seconds.
+     *
+     * @private
+     */
+    _seek: number,
+
+    /**
+     * Youtube id of the video to be played.
+     *
+     * @private
+     */
+    youtubeId: string
+};
+
+const YoutubeLargeVideo = (props: Props) => {
+    const playerRef = useRef(null);
+
+    useEffect(() => {
+        if (!props._isOwner) {
+            playerRef.current && playerRef.current.seekTo(props._seek);
+        }
+    }, [ props._seek ]);
+
+    useEffect(() => {
+        props._onWideScreenChanged(props._isWideScreen);
+    }, [ props._isWideScreen ]);
+
+    const onChangeState = e =>
+        playerRef.current && playerRef.current.getCurrentTime().then(time => {
+            if (props._isOwner && e !== 'buffering') {
+                props._onVideoChangeEvent(props.youtubeId, e, time, props._ownerId);
+            }
+        });
+    const onReady = () => {
+        if (props._isOwner) {
+            props._onVideoReady(
+                props.youtubeId,
+                playerRef.current && playerRef.current.getCurrentTime(),
+                props._ownerId);
+        }
+    };
+
+    let playerHeight, playerWidth;
+
+    if (props._isWideScreen) {
+        playerHeight = props._screenHeight;
+        playerWidth = playerHeight * 16 / 9;
+    } else {
+        playerWidth = props._screenWidth;
+        playerHeight = playerWidth * 9 / 16;
+    }
+
+    return (
+        <View
+            pointerEvents = { props._enableControls ? 'auto' : 'none' }
+            style = { styles.youtubeVideoContainer } >
+            <YoutubePlayer
+                height = { playerHeight }
+                initialPlayerParams = {{
+                    controls: props._enableControls,
+                    modestbranding: true,
+                    preventFullScreen: true
+                }}
+                /* eslint-disable react/jsx-no-bind */
+                onChangeState = { onChangeState }
+                /* eslint-disable react/jsx-no-bind */
+                onReady = { onReady }
+                play = { props._isPlaying }
+                playbackRate = { 1 }
+                ref = { playerRef }
+                videoId = { props.youtubeId }
+                volume = { 50 }
+                webViewProps = {{
+                    bounces: false,
+                    mediaPlaybackRequiresUserAction: false,
+                    scrollEnabled: false,
+                    userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36' // eslint-disable-line max-len
```

Comment:
IS this to prevent the Android play button? If so, can you please declare this as a constant at the top of the file and document it?